.github/workflows/hashlint.yml

```diff
@@ -0,0 +1,29 @@
+---
+name: Python lint workflow
+on:
+  pull_request:
+    paths:
+      - roles/jgrasp/vars/main.yml
+      - roles/eclipse/vars/main.yml
+      - roles/finch/vars/main.yml
+      - .github/workflows/hashlint.yml
+      - scripts/hashlint.py
+jobs:
+  Lint:
+    name: Run Lint tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Set up Python 3.x
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.x
+      - name: Install Python dependencies
+        run: |
+          pip3 install -U pyyaml requests
+      # We cannot just do roles/*/*/*py because we do not want to lint the
```

Comment:
This seems like it got copied from another file :)

Reply:
You caught me :)